src/main/java/seedu/address/model/person/Nric.java

```diff
@@ -10,8 +10,8 @@
 public class Nric {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "NRICs should contain 7 digits, with 2 capital letters at the beginning and end";
-    public static final String VALIDATION_REGEX = "\\p{Alpha}\\d{7}\\p{Alpha}";
+            "NRICs should contain 7 digits, with S or T at the beginning and a letter at the end";
+    public static final String VALIDATION_REGEX = "[ST][0-9]{7}[A-Za-z]";
```

Comment:
Can add lower case s and t also hahhaa ps

Reply:
done!